draft-packages/collapsible/KaizenDraft/Collapsible/styles.scss

```
@@ -67,10 +89,14 @@ $heading-active-color: $kz-var-color-wisteria-100;
   position: -webkit-sticky;
   position: sticky;
   z-index: 10;
+
+  .button {
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+  }
```

I don't think this will work since `.button` is not a descendant of `.sticky` (it's the same element). Maybe try removing the `.button` selector or try this

```suggestion
  &.button {
    border-top-left-radius: 0;
    border-top-right-radius: 0;
  }
```